### DISCUSSION TITLE
How can I delete drive?

### DISCUSSION THREAD

**Discussion by arguskao** [deta/cloud-docs#599] [Help]
I know create Drive.
But I do not know to delete/

Thanks

**Comment by sudo-subg** [comment]
You can delete the contents of a drive using drive [SDK](https://docs.deta.sh/docs/drive/sdk) and [HTTP API](https://docs.deta.sh/docs/drive/http). But to remove the drive completely you have to use [Drive UI](https://docs.deta.sh/docs/drive/drive_ui).

**Reply by arguskao** [reply]
Thanks.
